src/System.Data.SqlClient/src/System/Data/ProviderBase/DbConnectionPool.cs

```diff
@@ -413,10 +413,7 @@ internal WaitHandle[] GetHandles(bool withCreate)
 
             _objectList = new List<DbConnectionInternal>(MaxPoolSize);
 
-            if (ADP.IsPlatformNT5)
-            {
-                _transactedConnectionPool = new TransactedConnectionPool(this);
-            }
+            _transactedConnectionPool = new TransactedConnectionPool(this); // initialize irrespective of platform
```

Comment:
In AdapterUtil.cs `ADP.IsPlatformNT5` is not used anymore by any other part of the code. Can you remove these unused attributes as well?

Reply:
Done